### DISCUSSION TITLE
How to get frame rates and bit rates with video js and next js with version 8.12.0

### DISCUSSION THREAD

**Discussion by khushhalTravelxp** [videojs/video.js#8791] [Q&A]
### Description

Unable to get frame rates and bit rates . I have tried everything like player.tech().vhs and etc 

### Reduced test case

N/A

### Steps to reproduce

1. I want frame rates and bit rates with video js but not getting any reference .

### Errors

N/A

### What version of Video.js are you using?

8.12.0

### Video.js plugins used.

Nuevo 

### What browser(s) including version(s) does this occur with?

Chrome and video js version 8.12.0

### What OS(es) and version(s) does this occur with?

Window 11

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! 👋

If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md).
